thali/thalilogger.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var winston = require('winston');
+
+module.exports = function(){
+    return winston;
```

Comment:
Outputting a big number like a timestamp isn't terribly useful to someone reading the console. Also logcat automatically time stamps all of its output anyway (with a human readable string). So please remove the timestamp() from line 17 and then you can check this in!